nomenclature/processor/data_validator.py

```diff
@@ -86,9 +86,49 @@ def criteria(self):
         )
 
 
+class DataValidationCriteriaRange(DataValidationCriteria):
+    range: list[float] = Field(..., min_length=2, max_length=2)
+
+    @model_validator(mode="after")
+    def check_range_is_valid(self):
+        if self.range[0] > self.range[1]:
+            raise ValueError("Validation range is invalid: " + str(self.criteria))
+        return self
+
+    @computed_field
+    def upper_bound(self) -> float:
+        return self.range[1]
+
+    @computed_field
+    def lower_bound(self) -> float:
+        return self.range[0]
+
+    @property
+    def validation_args(self):
+        """Attributes used for validation (as bounds)."""
+        return self.model_dump(
+            exclude_none=True,
+            exclude_unset=True,
+            exclude=["warning_level", "range"],
+        )
+
+    @property
+    def criteria(self):
+        return self.model_dump(
+            exclude_none=True,
+            exclude_unset=True,
+            exclude=["warning_level", "lower_bound", "upper_bound"],
+        )
+
+
 class DataValidationCriteriaMultiple(IamcDataFilter):
     validation: (
-        list[DataValidationCriteriaValue | DataValidationCriteriaBounds] | None
+        list[
+            DataValidationCriteriaValue
```

Comment:
I would change this to:
```suggestion
            DataValidationCriteria
```